### ISSUE TITLE
Doesn't run at all?

### ISSUE THREAD

**Issue by ghost** [huchenlei/ComfyUI_DanTagGen#11]
I get an error: The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

And then nothing ever generates, it never moves, using example workflow:
![{A18B02D9-5271-438F-A047-F403A968F5B0}](https://github.com/user-attachments/assets/e22c1b72-e0a5-4bc2-b646-a8db3247d315)
went through all the setup of llama cpp and all but this isn't working ;_;
